ci_default.yml

```diff
@@ -38,3 +38,13 @@
   tasks:
     - name: Run partprobe
       shell: partprobe || true
+
+# WORKAROUND for https://bugzilla.redhat.com/show_bug.cgi?id=1456552#c5
+# TODO: delete after resolving
+- hosts: all
```

Comment:
I would suggest to use
`hosts: "usm_server:usm_nodes"`
Instead of `all`, because there might be situation, when inventory file contains additional hosts (for example in CentOS CI it contains also `qe_server`).